src/modules/formBuilder/formBuilderUtil.ts

```diff
@@ -1,12 +1,37 @@
 import { cloneDeep, kebabCase } from 'lodash-es';
+import { ItemMap } from '../form/types';
 import {
+  PickListOption,
   ItemType,
   Component,
   FormDefinitionJson,
   FormItem,
   Maybe,
 } from '@/types/gqlTypes';
 
+// Generate a list of Items to pick from. Used for picking dependent items for conditional logic.
+export const generateItemPickList = (
+  itemMap: ItemMap,
+  exclude: string[] = []
+) => {
+  const pickList: PickListOption[] = [];
+  Object.values(itemMap).forEach((item) => {
+    if (exclude.includes(item.linkId)) return;
+
+    let label = item.briefText || item.text || item.linkId;
```

Comment:
```suggestion
    let label = item.briefText || item.text || item.linkId;
    label = label.replace(/<[^>]*>/g, '');
```
?
We could also use a library like html-to-text to do this, but seems like overkill to me.

Reply:
Good call, added with https://github.com/greenriver/hmis-frontend/pull/788/commits/1f94b4ea62ad098b9b60f06dddd4b9971d761a2b